### DISCUSSION TITLE
[@types/braintree]  Validated Response doesn't have verification type

### DISCUSSION THREAD

**Discussion by rodelcabs** [DefinitelyTyped/DefinitelyTyped#67415] [Issues with a @types package]
`ValidatedResponse` interface doesn't have `verification` field for unsuccessful verification.

When doing a creditcard verification on `paymentMethod.create()`, if the verification failed, braintree returns the verification result object directly on response. as said in their documentation: [Link](https://developer.paypal.com/braintree/docs/guides/credit-cards/server-side/node#verification-results)
<img width="959" alt="image" src="https://github.com/DefinitelyTyped/DefinitelyTyped/assets/81414337/fdef2283-4adc-4508-8d0e-f46314a1b529">

However I am unable to handle this as ts is showing error: 
`verification does not exist on type ValidatedResponse<PaymentMethod>`
and checking the type code, ValidatedResponse does not have the verification field:
<img width="776" alt="image" src="https://github.com/DefinitelyTyped/DefinitelyTyped/assets/81414337/ff10b6b1-3d4a-4c12-9d96-9986163e2a90">

Am I missing something here? appreciate any help. Thanks

[a comment by an automated account is not shown]

**Comment by PaulOhNoes** [comment]
Bump. An update would be nice

**Comment by smrubin** [comment]
I no longer actively work with Braintree and will be removing myself as a module owner shortly.

This module has been largely community driven, if anyone wants to take a stab at implementing this!

I also would have thought / hoped that braintree would be providing typing directly from their sdk at this point, too.
